engine/settings/base.py

```diff
@@ -443,6 +478,10 @@ class BrokerTypes:
     "ALERT_GROUP_ESCALATION_AUDITOR_CELERY_TASK_HEARTBEAT_URL", None
 )
 
+SCHEDULE_FILENAME = os.getenv("SCHEDULE_FILENAME")
```

Comment:
I cannot use `CELERY_*` names because these variables are namespace by Celery library (see here: https://github.com/grafana/oncall/blob/dev/engine/engine/celery.py#L26-L30). So the only idea for this name is `BEAT_SCHEDULE_DB_FILENAME`, I'll prepare this change. If you have other ideas you are welcome :)

This option already has default value. If this variable isn't set, it will never be passed to Celery so the library will choose the internal default (`celerybeat-schedule`, in the working directory of the process).